C/c4Index.cc

```diff
@@ -67,29 +67,32 @@ C4IndexUpdater::~C4IndexUpdater() = default;
 
 size_t C4IndexUpdater::count() const {
     if ( hasFinished() ) {
-        WarnError("C4IndexUpdater::count() called on finished updater.");
+        Warn("C4IndexUpdater::count() called on finished updater.");
         return 0;
     }
     return _update->count();
 }
 
 FLValue C4IndexUpdater::valueAt(size_t i) const {
     if ( hasFinished() ) {
-        litecore::error::_throw(litecore::error::NotOpen, "C4IndexUpdater::valueAt() called on finished updater.");
+        Warn("C4IndexUpdater::valueAt() called on finished updater.");
+        return FLValue{};
```

Comment:
I don't know if this the same as null or not but there `kFLNullValue`.

Reply:
Here, FLValue is a pointer type. FLValue{} == nullptr